tests/common/tensor_test.py

```diff
@@ -238,7 +238,7 @@ def test_last_dim_softmax_does_softmax_on_last_dim(self):
 
     def test_last_dim_softmax_handles_mask_correctly(self):
         batch_size = 1
-        length_1 = 5
```

Comment:
I think this belonged with the prior tensor PR.  The reason was that there's an `expand_as` that works for the wrong reason if `num_options == length_1`, and it was masking an error that showed up in BiDAF.